AssetManagementTestProject/Common/AssetManagementTestBase.cs

```diff
@@ -0,0 +1,8 @@
+﻿using AssetManagementTestProject.TestSetup;
+
+namespace AssetManagementTestProject.Common;
+
+
+public class AssetManagementTestBase : NUnitWebTestSetup
```

Comment:
1. Remove empty line.
2. Add class purpose in a comment. The other cannot guess what you want to do

Reply:
Done!